pkg/satellite/satelliteauth/claims_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satelliteauth
+
+import (
+	"testing"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClaims(t *testing.T) {
+	id, err := uuid.New()
+	assert.Nil(t, err)
+	assert.NoError(t, err)
+	assert.NotNil(t, id)
+
+	claims := Claims{
+		ID:         *id,
+		Email:      "someEmail@ukr.net",
+		Expiration: time.Now(),
+	}
+
+	claimsBytes, err := claims.JSON()
+	assert.Nil(t, err)
+	assert.NoError(t, err)
```

Comment:
Aren't `assert.Nil(t, err)` and `assert.NoError(t, err)` redundant?

Reply:
@mobyvb yeah, it do the same, u're right